op-batcher/batcher/channel_manager.go

```diff
@@ -504,9 +504,8 @@ func (s *channelManager) Requeue(newCfg ChannelConfig) {
 	}
 
 	// We put the blocks back at the front of the queue:
-	s.blocks.Prepend(blocksToRequeue...)
-
 	for _, b := range blocksToRequeue {
+		s.blocks.Prepend(b)
```

Comment:
I assume this change is aesthetic (functionally equivalent to before)?

Reply:
Yes, I just wanted to harmonize the way we do this in the two different cases. We already need to loop over the blocks, so this way seems a bit nicer to me. Should be functionally equivalent.